### PR TITLE
Questions posting page visible when not logged-in

### LINKED ISSUE #1848: Make questions posting page visible when not logged-in, with friendly message
### Please describe the problem (or idea)

When you click on "ask a question" on the site, but you're not logged in, you see an error page:

https://publiclab.org/questions/new?title=lalalala

We should show the page with a nice prompt to log in or create an account first. The template for this page is here:

https://github.com/publiclab/plots2/tree/master/app/views/editor/questionRich.html.erb

Right now it's erroring because it depends on logged-in data; run it in development mode to see the exact errors!

Would love help with this one -- thank you!

### PR DESCRIPTION
**Pull request by Gauravano**
fixes #1848 
Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

### CODE REVIEW

**Comment by Gauravano** [comment]
This is the current screen displayed to people who would try to ask question while logged-out 

screenshot from 2017-12-12 22-39-41

In this, we can change text and generate customized notice instead.

**Comment by ebarry** [comment]
Wow great! 

Instead of "Please log in to access this page" could we change the text to "Your question is important and we want to hear from you! Please log in or sign up to post a question"

Also, could in "Public Lab network" could you please change it to "Public Lab community"?

**Comment by Gauravano** [comment]
Yes @ebarry, all changes would be done, will update you soon with updated page.Thank you.

**Comment by Gauravano** [comment]
This is the updated page view.Please suggest if any more changes are required.Thank you.

screenshot from 2017-12-13 02-39-07

**Comment by Gauravano** [comment]
@jywarren @ebarry please review the changes . Thank you.

File: app/controllers/questions_controller.rb [hunk 1]
```diff
@@ -28,10 +28,15 @@ def index
   end
 
   def new
-    if params[:legacy]
-      render 'editor/question'
-    else 
-      render 'editor/questionRich'
+    if current_user == nil
+      redirect_to '/login'
```
Comment (jywarren) [inline app/controllers/questions_controller.rb]: Hi! Can  you add `/login?return_to/questions/new` so you get sent back after logging in? Thanks!!!

File: app/views/editor/questionRich.html.erb [hunk 2]
```diff
@@ -100,7 +100,7 @@
     </div>
       
   </div>
-
+</div>>
```
[the remark this answers is not in the document]
Comment (jywarren) [reply app/views/editor/questionRich.html.erb]: Ooh, but watch out, there are two `>` here.

[the remark this answers is not in the document]
Comment (Gauravano) [reply app/views/editor/questionRich.html.erb]: correcting this too.thanks jeff.

**Comment by Gauravano** [comment]
@jywarren please suggest if any other changes are required.

**Comment by jywarren** [comment]
This looks super. Thanks so much for finding and fixing this! Grateful for your contributions!!

**Pull request merged**